openwisp_ipam/migrations/0001_initial.py

```diff
@@ -97,7 +97,7 @@ class Migration(migrations.Migration):
                         verbose_name='modified',
                     ),
                 ),
-                ('name', models.CharField(blank=True, db_index=True, max_length=100)),
+                ('name', models.CharField(db_index=True, max_length=100)),
```

Comment:
We will need a new migration for this change. We can only edit previous migration files for sample app.

Reply:
Since this is an application level change, we do not really need a new migration, because at database level nothing is changing, correct me if I am wrong.